application/src/main/java/org/ehrbase/application/abac/AbacConfig.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2021 Jake Smolka (Hannover Medical School) and Vitasystems GmbH.
+ *
+ * This file is part of project EHRbase
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehrbase.application.abac;
+
+import java.net.URI;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "abac")
+public class AbacConfig {
+
+  private URI server;
+  private String organizationClaim;
+  private String patientClaim;
+  // ehrStatus
+  @Value("${abac.policy.ehrstatus.name}")
```

Comment:
I would go either for `@ConfigurationProperties` either for `@Value` but having both looks strange to me.